lib/quickdraw/assertions.rb

```diff
@@ -47,6 +49,7 @@ def assert_equal_ruby(actual, expected)
 
 		assert(actual == expected) do
 			diff = DIFFER.diff_ruby(actual, expected)
+			diff = DIFFER.diff_strings(actual, expected) if no_syntactic_changes?(diff)
```

Comment:
If we'd use `diff_objects` here we'd also see the diff in whitespace, but I feel like it's not super nice to look at:

![CleanShot 2025-01-28 at 23 47 19@2x](https://github.com/user-attachments/assets/486accac-174b-429c-a10b-80db0a866c54)